tests/astronomer/flask_appbuilder/test_security.py

```diff
@@ -0,0 +1,152 @@
+from jwcrypto import jwk, jwt
```

Comment:
better to split to tests folder with conftest.py

Reply:
Conftest.py at top level, or alongside these tests?

Reply:
```
tree -L 1 tests
├── conftest.py
├── fixtures
├── test_dag_integrity.py
```

Reply:
Done, though mirroring the code structure of the app